webservices/soap/server/src/main/java/com/epam/soap/Application.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.soap;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import com.epam.soap.config.AppConfig;
+
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        SpringApplication.run(new Class<?>[] {AppConfig.class}, args);
```

Comment:
They have `run` method with `Object` (not array) as first argument, looks prettier :)